packages/canary-react/src/stories/ic-date-picker.stories.mdx

```diff
@@ -506,4 +508,109 @@ const DatePickerWithIcChange = () => {
 };
 
 export default DatePickerWithIcChange;
-```
\ No newline at end of file
+```
+
+### Playground example
+
+Go to the <IcLink href="/?path=/story/react-components-date-picker--playground-example">separate page for the playground example</IcLink> to view the prop controls.
+
+export const defaultArgs = {
+  dateFormat: "DD/MM/YYYY",
+  disabled: false,
+  disableDays: [],
+  disableDaysMessage: "This day is not available",
+  disableFuture: false,
+  disableFutureMessage: "This date is in the future",
+  disablePast: false,
+  disablePastMessage: "This date is in the past",
+  helperText: "Choose a date",
+  label: "When would you like to collect your coffee?",
+  max: "",
+  min: "",
+  openAtDate: "",
+  required: false,
+  showDaysOutsideMonth: true,
+  showPickerClearButton: true,
+  showPickerTodayButton: true,
+  size: "default",
+  startOfWeek: 1,
+  validationStatus: "",
+  validationText: "",
+  value: "",
+};
+
+<Canvas withSource="none">
+  <Story
+    args={defaultArgs}
+    argTypes={{
+      dateFormat: {
+        options: ["DD/MM/YYYY", "MM/DD/YYYY", "YYYY/MM/DD"],
+        control: {
+          type: "radio",
+        },
+      },
+      validationStatus: {
+        options: ["", "error", "success", "warning"],
+        control: {
+          type: "radio",
+        },
+      },
+      size: {
+        options: ["small", "default", "large"],
+        control: {
+          type: "radio",
+        },
+      },
+      startOfWeek: {
+        options: [0, 1, 2, 3, 4, 5, 6],
+        control: {
+          type: "select",
+        },
+      },
+      openAtDate: {
+        control: {
+          type: "text",
+        },
+      },
+    }}
+    name="Playground example"
+  >
+    {(args) => {
+      const [{ value }, updateArgs] = useArgs();
+
+      const updateDate = (ev) => {
+        const date = ev.detail.value;
+        const formattedDate = `${date.getDate()}-${date.getMonth() + 1}-${date.getFullYear()}`;
+        updateArgs({ value: formattedDate});
```

Comment:
sorry, could this also handle setting an empty string when the value is cleared?